.dockerignore

```diff
@@ -0,0 +1 @@
+*/.env*
```

Comment:
nitpick : in visual studio you can check this so that newlines will come after save
![image](https://user-images.githubusercontent.com/8665738/148070746-cb3706e4-d076-42f6-8a5c-5aefc6727c88.png)

Reply:
yes but we should have it in lint config

Reply:
i'm going to create a new PR & a git hook for this separately to make it easier & seamless regardless of ide/env